app/src/atoms/Banner/index.tsx

```diff
@@ -88,23 +90,33 @@ export function Banner(props: BannerProps): JSX.Element {
     ...styleProps
   } = props
   const bannerProps = BANNER_PROPS_BY_TYPE[type]
-
   const iconProps = {
     ...(icon ?? bannerProps.icon),
     size: size ?? SIZE_1,
     marginRight: iconMarginRight ?? SPACING.spacing3,
     marginLeft: iconMarginLeft ?? '0rem',
     color: BANNER_PROPS_BY_TYPE[type].color,
   }
+  const ON_DEVICE_BANNER_STYLE = css`
+    border: ${String(SPACING.spacingXXS)} ${String(BORDERS.styleSolid)}
+      ${BANNER_PROPS_BY_TYPE[type].color};
+    font-size: ${TYPOGRAPHY.fontSizeP};
+    font-weight: ${TYPOGRAPHY.fontWeightRegular};
+    border-radius: ${SPACING.spacing2};
+
+    @media ${ODD_MEDIA_QUERY_SPECS} {
+      font-size: 1.25rem;
+      font-weight${TYPOGRAPHY.fontWeightSemiBold};
+      border: 0rem;
+      background-color: #FFE1A4;
```

Comment:
```suggestion
      background-color: ${COLORS.yellow_three};
```

if you merge edge ☝️should work

Reply:
whoops completely forgot I added these colors to `banner`, I will go back and use the constants 😄